.github/workflows/wardend.yaml

```
@@ -0,0 +1,88 @@
+    paths:
+      - "cmd/wardend/**"
+      - "warden/**"
+      - "precompiles/**"
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    name: lint
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-go@v4
```

actionlint (1.7.4)

the runner of "actions/setup-go@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)